source/upgrade.txt

```diff
@@ -104,6 +104,16 @@ Version 5.0 Breaking Changes
      
      VectorSearchOptions opts = vectorSearchOptions().filter(eq("x", 8));
 
+- This driver removes the
+  ``org.mongodb.scala.ObservableImplicits.ToSingleObservableVoid`` implicit
+  class, which means that ``org.reactivestreams.Publisher[Void]`` is no longer
+  automatically convertible to ``org.mongodb.scala.SingleObservable[Void]``. The
```

Comment:
Q: "automatically" = "implicitly" ?

Reply:
Ya, using an implicit class means the conversion can happen automatically. Devs should know this, but I've rephrased this to make it clear.